soc/arm/st_stm32/stm32f0/dts_fixup.h

```diff
@@ -1,60 +1,114 @@
+#define CONFIG_GPIO_STM32_GPIOB_CLOCK_BITS        ST_STM32_GPIO_48000400_CLOCK_BITS
+#define CONFIG_GPIO_STM32_GPIOB_CLOCK_BUS         ST_STM32_GPIO_48000400_CLOCK_BUS
+
+#define CONFIG_GPIO_STM32_GPIOC_BASE_ADDRESS      ST_STM32_GPIO_48000800_BASE_ADDRESS
+#define CONFIG_GPIO_STM32_GPIOC_CLOCK_BITS_0      ST_STM32_GPIO_48000800_CLOCK_BITS_0
+#define CONFIG_GPIO_STM32_GPIOC_CLOCK_BUS_0       ST_STM32_GPIO_48000800_CLOCK_BUS_0
+#define CONFIG_GPIO_STM32_GPIOC_CLOCK_CONTROLLER  ST_STM32_GPIO_48000800_CLOCK_CONTROLLER
+#define CONFIG_GPIO_STM32_GPIOC_LABEL             ST_STM32_GPIO_48000800_LABEL
+#define CONFIG_GPIO_STM32_GPIOC_SIZE              ST_STM32_GPIO_48000800_SIZE
+#define CONFIG_GPIO_STM32_GPIOC_CLOCK_BITS        ST_STM32_GPIO_48000800_CLOCK_BITS
+#define CONFIG_GPIO_STM32_GPIOC_CLOCK_BUS         ST_STM32_GPIO_48000800_CLOCK_BUS
+
+#define CONFIG_GPIO_STM32_GPIOD_BASE_ADDRESS      ST_STM32_GPIO_48000C00_BASE_ADDRESS
+#define CONFIG_GPIO_STM32_GPIOD_CLOCK_BITS_0      ST_STM32_GPIO_48000C00_CLOCK_BITS_0
+#define CONFIG_GPIO_STM32_GPIOD_CLOCK_BUS_0       ST_STM32_GPIO_48000C00_CLOCK_BUS_0
+#define CONFIG_GPIO_STM32_GPIOD_CLOCK_CONTROLLER  ST_STM32_GPIO_48000C00_CLOCK_CONTROLLER
+#define CONFIG_GPIO_STM32_GPIOD_LABEL             ST_STM32_GPIO_48000C00_LABEL
+#define CONFIG_GPIO_STM32_GPIOD_SIZE              ST_STM32_GPIO_48000C00_SIZE
+#define CONFIG_GPIO_STM32_GPIOD_CLOCK_BITS        ST_STM32_GPIO_48000C00_CLOCK_BITS
+#define CONFIG_GPIO_STM32_GPIOD_CLOCK_BUS         ST_STM32_GPIO_48000C00_CLOCK_BUS
+
+#define CONFIG_GPIO_STM32_GPIOE_BASE_ADDRESS      ST_STM32_GPIO_48001000_BASE_ADDRESS
+#define CONFIG_GPIO_STM32_GPIOE_CLOCK_BITS_0      ST_STM32_GPIO_48001000_CLOCK_BITS_0
+#define CONFIG_GPIO_STM32_GPIOE_CLOCK_BUS_0       ST_STM32_GPIO_48001000_CLOCK_BUS_0
+#define CONFIG_GPIO_STM32_GPIOE_CLOCK_CONTROLLER  ST_STM32_GPIO_48001000_CLOCK_CONTROLLER
+#define CONFIG_GPIO_STM32_GPIOE_LABEL             ST_STM32_GPIO_48001000_LABEL
+#define CONFIG_GPIO_STM32_GPIOE_SIZE              ST_STM32_GPIO_48001000_SIZE
+#define CONFIG_GPIO_STM32_GPIOE_CLOCK_BITS        ST_STM32_GPIO_48001000_CLOCK_BITS
+#define CONFIG_GPIO_STM32_GPIOE_CLOCK_BUS         ST_STM32_GPIO_48001000_CLOCK_BUS
+
+#define CONFIG_GPIO_STM32_GPIOF_BASE_ADDRESS      ST_STM32_GPIO_48001400_BASE_ADDRESS
+#define CONFIG_GPIO_STM32_GPIOF_CLOCK_BITS_0      ST_STM32_GPIO_48001400_CLOCK_BITS_0
+#define CONFIG_GPIO_STM32_GPIOF_CLOCK_BUS_0       ST_STM32_GPIO_48001400_CLOCK_BUS_0
+#define CONFIG_GPIO_STM32_GPIOF_CLOCK_CONTROLLER  ST_STM32_GPIO_48001400_CLOCK_CONTROLLER
+#define CONFIG_GPIO_STM32_GPIOF_LABEL             ST_STM32_GPIO_48001400_LABEL
+#define CONFIG_GPIO_STM32_GPIOF_SIZE              ST_STM32_GPIO_48001400_SIZE
+#define CONFIG_GPIO_STM32_GPIOF_CLOCK_BITS        ST_STM32_GPIO_48001400_CLOCK_BITS
+#define CONFIG_GPIO_STM32_GPIOF_CLOCK_BUS         ST_STM32_GPIO_48001400_CLOCK_BUS
+
+#define CONFIG_UART_STM32_USART_1_BASE_ADDRESS  ST_STM32_USART_40013800_BASE_ADDRESS
```

Comment:
Looks like the whitespace between the things is getting to spaces vs tabs.  We've tried to use tabs, can you see if you can maintain that.